padd.sh

```diff
@@ -497,10 +473,9 @@ GetVersionInformation() {
     today=$(date +%Y%m%d)
 
     # was the last check today?
-    if [ "${today}" != "${last_check}" ]; then # no, it wasn't today
-      # Remove the Pi-hole version file...
-      rm -f piHoleVersion
-    fi
+    # No, because: last_check is referenced but not assigned. [SC2154] - removed.
```

Comment:
Do not remove that code block `last_check` is read from `./piHoleVersion`

Reply:
Has been inserted again.